addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -45,7 +45,7 @@
                         <div t-attf-class="mb-3 row o_url_input#{state.isButton ? ' d-none' : ''}">
                             <label class="col-form-label col-md-3" for="o_link_dialog_url_input">URL or Email</label>
                             <div class="col-md-9">
-                                <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required" t-ref="inputUrl" t-on-keydown="ev => ev.key === 'Enter' &amp;&amp; this.onSave(ev)"/>
+                                <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required" t-ref="inputUrl" t-on-keydown="ev => this.onKeyDown(ev)"/>
```

Comment:
```suggestion
                                <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required" t-ref="inputUrl" t-on-keypress="ev => ev.key === 'Enter' &amp;&amp; this.onSave(ev)"/>
```
Just changing `t-on-keydown` to `t-on-keypress` will work here. :clinking_glasses: